src/main/java/com/jcabi/aether/Aether.java

```diff
@@ -94,10 +94,6 @@
  *  using jcabi-http, provide a maven configuration file with proxy settings
  *  and verify the expected HTTP requests have been sent to the mocked server.
  * @todo #69:30min Fix UnsupportedClassVersionError in CL build for Open JDK6.
- * @todo #82:30min Clean unnecessary docker setup (src/docler/Dockerfile),
- *  make the configuration minimal. Set versions to the used packages
- *  (e.g. lock java version instead of default jdk). After changes, the file
- *  needs to be uploaded to Dockerhub and be built there.
```

Comment:
@amihaiemil the thing is that @yegor256 always wanted PR to https://github.com/yegor256/dockers and then he was uploading... @original-brownbear do you remember why?